### PR TITLE
Add changelog entry for prototype kit config build

### PR DESCRIPTION
**Pull request by colinrotherham**
This PR adds a changelog entry for:

* https://github.com/alphagov/govuk-frontend/pull/2851

Closes https://github.com/alphagov/govuk-frontend/issues/2889

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -185,6 +185,8 @@ This was introduced in [pull request #2844: Remove compatibility mode from govuk
 
 ### Fixes
 
+In [pull request 2851: Supporting v13 of the Prototype Kit](https://github.com/alphagov/govuk-frontend/pull/2851) we made `govuk-prototype-kit.config.json` part of our package build. The updated config adds JavaScript to initialise GOV.UK Frontend via the Prototype Kit plugin interface, including a new list of component macros and their import paths.
```
Comment (36degrees) [inline CHANGELOG.md:188]: I'm not sure it's that useful for users to know how this works in this much detail, and arguably this doesn't really tell them what `govuk-prototype-kit.config.json` actually _does_.

I wonder if we should simplify to something like

> In [pull request 2851: Supporting v13 of the Prototype Kit](https://github.com/alphagov/govuk-frontend/pull/2851) we introduced support for the plugins system introduced in the upcoming Prototype Kit v13.

It's a bit repetitive given the PR title…. I wonder if we could just change the title of the PR to 'Support Prototype Kit v13' and then just list this with the rest of the fixes below?

@claireashworth do you have any thoughts on this one?

**Comment by 36degrees** [comment]
Removed this from the milestone as the linked issue is already associated (so it's effectively represented in the milestone twice)

File: CHANGELOG.md (hunk 1 above)
Comment (colinrotherham) [reply CHANGELOG.md]: @36degrees Yeah that's sensible, I've pushed up your first suggestion

At least two other `### Fixes` sections included the format:

```md
In [pull request XXXX: Description…]
```

So I pinched it 😊 

If the fixes list is a better place, let me know

In terms of what it does, I knew [this bit was further down](https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#270-feature-release). If it helps at all?

```md
## 2.7.0 (Feature release)

🆕 New features:

- Added config file for prototype kit.

  Added a configuration file for an experimental feature in Prototype kit to allow departments to use their own frontend alongside govuk-frontend

  Thanks @matcarey (HMRC) for this contribution

  ([PR #1102](https://github.com/alphagov/govuk-frontend/pull/1102))
```

Cheers for the milestone tip 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -185,6 +185,8 @@ This was introduced in [pull request #2844: Remove compatibility mode from govuk
 
 ### Fixes
 
+In [pull request 2851: Support Prototype Kit v13](https://github.com/alphagov/govuk-frontend/pull/2851) we introduced support for the plugins system introduced in the upcoming Prototype Kit v13.
```
Comment (claireashworth) [inline CHANGELOG.md:188]:
```suggestion
In [pull request 2851: Support Prototype Kit v13](https://github.com/alphagov/govuk-frontend/pull/2851) we introduced support for the plugins system included in the upcoming Prototype Kit v13.
```

Comment (colinrotherham) [reply CHANGELOG.md]: Nice spot, double "introduced"! Pushed it up

@36degrees @claireashworth Good to go?

**Review by 36degrees — APPROVED** [review]

**Review by claireashworth — APPROVED** [review]

**Pull request merged**